docs_app/README.md

```diff
@@ -80,7 +80,7 @@ You also want to see those changes displayed properly in the doc viewer
 with a quick, edit/view cycle time.
 
 For this purpose, use the `npm run docs-watch` task, which watches for changes to source files and only
-re-processes the the files necessary to generate the docs that are related to the file that has changed.
+re-processes the files necessary to generate the docs that are related to the file that has changed.
```

Comment:
I've only recently started flagging doubled words and am still not terribly confident about these fixes.